submissions/korotushak96/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener('DOMContentLoaded', ()=>{
```

Comment:
Respect for using `DOMContentLoaded`!
You can make some performance optimization for your app, but, of course, this is up to you.

This fix requires creating some function called `initAnimalsApp`, for example. And put there all your script.
Then, instead of an anonymous function, I recommend you to create a name for it (`initHandler`, for example) and add this handler for the `DOMContentLoaded` event listener.
After calling `initAnimalsApp` you could remove event listener with `initHandler`.

Reply:
thnaks for idea. I'll try to implement it